source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -39,21 +39,98 @@ You can create a Prometheus service within your PaaS space and allow it to bind
 
 Because changes to the first route will only be picked up within 5 minutes it could create gaps to metrics during route changes if you're using a zero downtime plugin such as [autopilot][].
 
+## Configure container metrics
+
+[Cloud Foundry][] provides time-series data (metrics), for your PaaS apps.
+
+Use the metrics exporter app to send Cloud Foundry metrics data to a monitoring system of your choice to store and view it. You'll need to deploy the metrics exporter as an app on PaaS before you can use it.
+
+Currently suppprted metrics are:
+
+* CPU
+* RAM
+* disk usage data
+* app crashes
+* app requests
+* app response times
+
+### Set up the metrics exporter app
+
+Before you setting up the metrics exporter app, you'll need a live Cloud Foundry account assigned to the spaces you want to receive metrics on.
+
+Your new account should be separate to your primary Cloud Foundry account and use the [`SpaceAuditor`][] role beause it can view app data without modifying it.
+
+To set up the metrics exporter app:
+
+1. Clone the [paas-metric-exporter][] GitHub repository.
+2. [Push the metrics exporter app](https://docs.cloud.service.gov.uk/#deployment-overview) to Cloud Foundry (without starting the app) by running
+ * `cf push --no-start metric-exporter`
+3. Set the following mandatory environment variables in the metrics exporter app using
+  * `cf set-env metric-exporter NAME VALUE`
+
+	Use the `cf set-env` command for these mandatory variables, this will keep secure the secret information contained in them.
+
+	|Name|Value|
+	|:---|:---|
+	|`API_ENDPOINT`|Use `https://api.cloud.service.gov.uk`|
+	|`USERNAME`|Cloud Foundry user|
+	|`PASSWORD`|Cloud Foundry password|
+	|`ENABLE_STATSD`|false|
+	|`ENABLE_PROMETHEUS`|true|
+
+	You could also set environment variables by amending the manifest file for optional environment variables that do not contain secret information. Read [paas-metric-exporter][] GitHub repository for more information.
+
+4. Run `cf start metric-exporter` to start your app
+1. Check you're generating Prometheus metrics at the metrics endpoint
+    - `https://<your-app-url>/metrics`
+5. Bind your app to the Prometheus service
+  - `cf bind-service <app-name> <service-instance-name>`
+
+### IP whitelist your app
+
+1. Register the IP whitelist route service as a user-provided service in your PaaS space.
+
+    `cf create-user-provided-service re-ip-whitelist-service -r https://re-ip-whitelist-service.cloudapps.digital`
+
+
+2. Register the route service for routes you want to protect.
+
+    `cf bind-route-service cloudapps.digital re-ip-whitelist-service --hostname <your paas app route>`
+
+    For example, `app-to-protect.cloudapps.digital` would be:
+
+    `cf bind-route-service cloudapps.digital re-ip-whitelist-service --hostname app-to-protect`
+
+### Troubleshooting
+
+If you're not receiving metrics, check the [logs][] for the metrics exporter app or contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk][].
```

Comment:
is there a Slack channel people can use?

Reply:
We should also be clear if it's re-tools or paas team supporting this. I don't think we should put the paas support email address on here unless we've asked them?

Reply:
Should I remove that email and add our slack channel?

Reply:
yes, let's do that